partcad/src/partcad/shape.py

```
@@ -223,10 +234,7 @@
                 line_weight = 1.0
 
         if viewport_origin is None:
-            if (
-                "viewportOrigin" in svg_opts
-                and not svg_opts["viewportOrigin"] is None
-            ):
+            if "viewportOrigin" in svg_opts and not svg_opts["viewportOrigin"] is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)